### ISSUE TITLE
[BUG] File spec loading error in legacy project

### ISSUE THREAD

**Issue by pafmon** [oas-tools/oas-tools#362]
## **Describe the bug**
When updating the oas-tools package version to 3, there is a spec loading error

## **To Reproduce**
1. Clone the "oas-error" branch in the ks-api repo: https://github.com/pafmon/ks-api/tree/oas-error
2. Try to run it with: npm install ; npm start

```
> ks-api@1.1.0 start
> node index.js

2023-01-25 10:01:81 [oas-tools] WARN: This initialization is deprecated and will be removed in the future. Please read the docs.
2023-01-25 10:01:82 [oas-tools] WARN: Ignored ./.dockerignore when searching for OASDoc location. Could't read file or directory
2023-01-25 10:01:82 [oas-tools] ERROR: ValidationError: Specification file at /home/pablofm/devel/github/ks-api/api/oas-file.yaml not found
    at validateOASFile (/home/pablofm/devel/github/ks-api/node_modules/@oas-tools/commons/dist/utils/modules/validator.js:13:15)
    at file:///home/pablofm/devel/github/ks-api/node_modules/oas-tools/src/index.js:37:7
    at async initialize (file:///home/pablofm/devel/github/ks-api/node_modules/oas-tools/src/index.js:31:5)
    at async file:///home/pablofm/devel/github/ks-api/node_modules/oas-tools/src/index_old.js:41:9
npm notice
npm notice New major version of npm available! 8.19.3 -> 9.3.1
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.3.1
npm notice Run npm install -g npm@9.3.1 to update!
npm notice
```
If I try to rename the file to ``oas-file.yaml``, another error occurs:
```
> npm start

> ks-api@1.1.0 start
> node index.js

node:internal/fs/utils:348
    throw err;
    ^

Error: ENOENT: no such file or directory, open '/home/pablofm/devel/github/ks-api/api/oas-doc.yaml'
    at Object.openSync (node:fs:600:3)
    at Object.readFileSync (node:fs:468:35)
    at Object.<anonymous> (/home/pablofm/devel/github/ks-api/index.js:20:15)
    at Module._compile (node:internal/modules/cjs/loader:1218:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
    at Module.load (node:internal/modules/cjs/loader:1081:32)
    at Module._load (node:internal/modules/cjs/loader:922:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/home/pablofm/devel/github/ks-api/api/oas-doc.yaml'
}

Node.js v18.13.0
```

**Comment by alesancor1** [comment]
Related to #357

**Comment by alesancor1** [comment]
This issue has been fixed at 8bb0aa5 and will be closed once the new version is released.
